R/ff16w.R

```diff
@@ -64,16 +64,17 @@ FF16w_StochasticPatchRunner <- function(p) {
 ## Helper:
 ##' @export
 ##' @rdname FF16_Environment
-##' @param infil_rate rate of water entering the first layer
-##' @param n_layers the number of layers
-##' @param init starting conditions
+##' @param soil_number_of_depths the number of soil layers
+##' @param rainfall_x list of x (time) positions for rainfall spline
+##' @param rainfall_y list of y (rainfall) positions for rainfall spline
 FF16w_make_environment <- function(canopy_light_tol = 1e-4, 
                                    canopy_light_nbase = 17,
                                    canopy_light_max_depth = 16, 
                                    canopy_rescale_usually = TRUE,
                                    soil_number_of_depths = 1,
                                    soil_initial_state = 0.0,
-                                   soil_infiltration_rate = 0.0) {
+                                   rainfall_x = seq(0, 9, 1),
```

Comment:
You're right that need x & y. I would rather pass in a single data frame than an array, so perhaps change this accordingly?

Reply:
So single data frame for `FF15w_make_environment` which is split into two arrays to pass into `rainfall_add_points()`?